checker/src/main/java/org/checkerframework/checker/nullness/NullnessTransfer.java

```diff
@@ -351,6 +352,14 @@ public TransferResult<NullnessValue, NullnessStore> visitMethodAccess(
     @Override
     public TransferResult<NullnessValue, NullnessStore> visitFieldAccess(
             FieldAccessNode n, TransferInput<NullnessValue, NullnessStore> p) {
+        if (n.getFieldName().equals("class")) {
```

Comment:
As described in #548 we should look into properly representing class literals in the CFG. The description for `FieldAccessNode` explicitly states that they are not for class literals. Let's figure out what a good representation for this is.